Models/MetAPI/GraphQLTypes/ForecastType.cs

```diff
@@ -12,6 +15,43 @@ public ForecastType()
             Field(forecast => forecast.ForecastProperties, false, typeof(PropertiesType))
                 .Description("Data and meta-data returned");
             Field(forecast => forecast.Type);
+
+            // What the actual metadata should look like, is hopefully compatible with most databases 
+            Field(forecast => forecast.StoredMetaData, false, typeof(StoredMetaDataType));
+
+            // Very bad and temporarily workaround connecting to a Atlas Cluster containing some metadata documents 
+            // connect using MongoClient
+            var dbClient =
+                new MongoClient(
+                    "mongodb+srv://new_user_31:<password>@cluster0.3cgl6.mongodb.net/metadata?retryWrites=true&w=majority");
```

Comment:
This connection string should be read from the configuration, akin to the `MOCK` variable. This makes it possible to deploy this to Azure and store the DB credentials there.

Reply:
It also makes it easier to support dev/prod DB.

Reply:
String is now read from appsettings._Launch Profile_.json